docs/src/routes/docs/components/form-field.svx

```diff
@@ -0,0 +1,121 @@
+---
+name: FormField
+---
+
+<script>
+  import { FormField, TextField, Switch } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+
+  let happy = false;
+</script>
+
+
+# FormField
+
+A responsive form field skeleton – providing a way to label and describe the field.
+
+Note that this component doesn't add any form handling capabilities; it is merely presentational.
+
+<Showcase>
+  <div slot="showcase">
+    <FormField
+      name="Main field"
+      help="Write this and not that."
+      required
+    >
+      <TextField />
+    </FormField>
+    <FormField
+      name="Secondary field"
+      optional
+    >
+      <TextField />
+    </FormField>
+    <FormField
+      name="Happiness"
+      errors={[happy && 'Not happy enough']}
+    >
+      <Switch bind:value={happy} />
+    </FormField>
+  </div>
+  <div slot="source">
+
+```svelte
+<script>
+  let happy = false;
+</script>
+
+<FormField
+  name="Main field"
+  help="Write this and not that."
+  required
+>
+  <TextField />
+</FormField>
+<FormField
+  name="Secondary field"
+  optional
+>
+  <TextField />
+</FormField>
+<FormField
+  name="Happiness"
+  errors={[happy && 'Not happy enough']}
+>
+  <Switch bind:value={happy} />
+</FormField>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`name`** | `null` | `string` | The name of the form field. Displayed prominently next to the actual field. |
+| **`help`** | `null` | `string` | The subtitle text under the name providing extra guidance. |
+| **`id`** | `null` | `string` | The ID to add the the `for` attribute of the `<label>` element containing the ==`name`==. Useful if you pass an ID to the actual field inside and want to connect it with the label. |
+| **`required`** | `false` | `boolean` | Marks the form field as required, adding an asterisk to the name and text under the field indicating that the field is mandatory. |
+| **`optional`** | `false` | `boolean` | Marks the form field as optional, explicitly stating the optionality of the field in the text under the field. |
+| **`errors`** | `null` | `Array` | An array of error messages to display under the field. Falsy values disregarded. |
```

Comment:
For similar reasoning as the `null` vs `Infinity` before, isn't `[]` a more reasonable default value here?

Reply:
Yeah, probably